apm-sniffer/apm-sdk-plugin/nats-client-plugin/src/main/resources/skywalking-plugin.def

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+nats-client-2.14.x-2.15.x=org.apache.skywalking.apm.plugin.nats.client.define.NatsMessageInstrumentation
```

Comment:
You declared versions range here, so I think you should update the plugin folder name and scenario folder name to adopt the versions too?

Reply:
Could I choose remove this declared ?

Reply:
I think you should keep versions. Most plugins include the versions in the plugin name, which makes users easily to choose.

Reply:
Got it .